spec/ruby-mpd/plugins/controls_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'spec_helper'
+require_relative '../../../lib/ruby-mpd/plugins/controls'
+
+RSpec.describe MPD::Plugins::Controls do
+  class MPD
+    def send_command(command, *args); end
+    def priority(); end
+  end
+
+  subject { MPD.new.extend described_class }
+
+  context "#next" do
+    it "should send correct params" do
+      expect(subject).to receive(:send_command).with(:next)
+      subject.next
+    end
+  end
+
+  context "#pause=" do
+    it "should send correct params" do
+      expect(subject).to receive(:send_command).with(:pause, 'yes')
+      subject.pause = 'yes'
```

Comment:
To start with I have ignored the comments and just tested the method in question as I see them. So in this example the method is:

```
def pause=(toggle)
  send_command :pause, toggle
end
```

Which does not appear to care what gets passed in, and just takes the argument and passes it on to the send_command method. I guess if toggle has to be a boolean, the method will need refactoring to handle *just* booleans.

So lets imagine you refactored the method to *only* accept booleans. The test would then fail and it would have to be updated to support the more specific requirement.